Core/Core/Common/CommonUI/UIViews/DynamicButton.swift

```diff
@@ -70,22 +70,18 @@ open class DynamicButton: UIButton {
         }
     }
 
-    open override var isHighlighted: Bool {
-        didSet {
-            tintColor = titleColor(for: state)
-        }
+    required public init?(coder: NSCoder) {
+        super.init(coder: coder)
```

Comment:
This initializer is invoked when the button is instantiated from storyboard. Shouldn't we register for trait changes from here as well?